Etabs_Adapter/Types/PriorityTypes.cs

```diff
@@ -0,0 +1,78 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2024, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Structure.MaterialFragments;
+using BH.oM.Structure.Elements;
+using BH.oM.Structure.SectionProperties;
+using BH.oM.Structure.SurfaceProperties;
+using BH.oM.Structure.Constraints;
+using BH.oM.Structure.Loads;
+using System;
+using BH.Engine.Adapter;
+using BH.oM.Adapters.ETABS;
+using System.Collections.Generic;
+using BH.oM.Spatial.SettingOut;
+
+namespace BH.Adapter.ETABS
+{
+#if Debug16 || Release16
+    public partial class ETABS2016Adapter : BHoMAdapter
+#elif Debug17 || Release17
+   public partial class ETABS17Adapter : BHoMAdapter
+#else
+    public partial class ETABSAdapter : BHoMAdapter
+#endif
+    {
+        /***************************************************/
+        /**** Protected methods                         ****/
+        /***************************************************/
+
+        protected void SetupPriorities()
+        {
+
+            PriorityTypes = new Dictionary<Type, int>
+            {
+                {typeof(Level),0 },
+                {typeof(Node),1 },
+                {typeof(ISectionProperty),2 },
+                {typeof(IMaterialFragment),3 },
+                {typeof(Bar),4 },
+                {typeof(ISurfaceProperty),5 },
+                {typeof(Panel),6 },
+                {typeof(RigidLink),7 },
+                {typeof(LinkConstraint),8 },
+                {typeof(Loadcase),9 },
+                {typeof(IElementLoad<Node>),10 },
+                {typeof(IElementLoad<Bar>),11 }
+            };
+
```

Comment:
With suggested changes to the BHoM_Adapter, this can simply be this. Think this should not be instead of the dependency types, but just for exceptional cases where that is not enough. 

```suggestion
            PriorityTypes = new List<Type>
            {
                {typeof(Level)},
            };
```

Reply:
@IsakNaslundBh , thanks for the suggestion.
Incorporated and commited.